<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicPage</class>
 <widget class="QWidget" name="BasicPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>494</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label1_alias">
       <property name="text">
        <string>Alias*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_alias</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lrcfg_alias">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Anything, used only to select the account</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label2_protocol">
       <property name="text">
        <string>Protocol</string>
       </property>
       <property name="buddy">
        <cstring>m_pProtocol</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="AutoComboBox" name="m_pProtocol"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Display Name*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_displayName</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lrcfg_displayName">
       <property name="placeholderText">
        <string>The name as seen by the peer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lrcfg_registeredNameBuddy2">
       <property name="text">
        <string>Re&amp;gistered Name*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_registeredName</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lrcfg_registeredName">
         <property name="placeholderText">
          <string>The name seen by your peers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lrcfg_registeredNameBuddy">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label3_server">
       <property name="text">
        <string comment="SIP server">Server*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_hostname</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lrcfg_hostname">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Example: sip.sring.cx or 192.168.0.2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label4_user">
       <property name="accessibleDescription">
        <string comment="Account username"/>
       </property>
       <property name="text">
        <string comment="Account username">&amp;Username*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_username</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lrcfg_username">
       <property name="placeholderText">
        <string>Usually your extension number</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label5_password">
       <property name="text">
        <string comment="Account password">Password*</string>
       </property>
       <property name="buddy">
        <cstring>lrcfg_password</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lrcfg_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>SIP password, ask your network administrator</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="lrcfg_hasCustomBootstrap">
     <property name="text">
      <string>Use custom bootstrap nodes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="m_pBootstrapModel">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="lrcfg_autoAnswer">
     <property name="text">
      <string>Auto answer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label7_state">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="edit7_state">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pDisplayLog">
       <property name="text">
        <string>Availability log</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-information">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/autocombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lrcfg_hasCustomBootstrap</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pBootstrapModel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
